options/locale/locale_en-US.ini

```diff
@@ -1746,6 +1746,8 @@ wiki.create_first_page = Create the First Page
 wiki.page = Page
 wiki.filter_page = Filter page
 wiki.new_page = Page
+wiki.page_title = Page Title
+wiki.page_content = Page Content
```

Comment:
Here again?

Reply:
I think we should clearly define where we use title case, and where we don't use it.

Reply:
Yeah, some guideline will be good for this in the future.